service/middleware.go

```diff
@@ -417,13 +417,13 @@ func ForbiddenHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 // Logger is a middleware handler that logs the request as it goes in and the response as it goes out.
-type Logger struct {
+type RecoveryLogger struct {
 	// ids is a channel producing unique, autoincrementing request ids that are included in logs.
 	ids chan int
 }
 
 // NewLogger returns a new Logger instance
-func NewLogger() *Logger {
+func NewRecoveryLogger() *RecoveryLogger {
```

Comment:
Fix comment. Also, fix the test in api_middleware_test.go.

Reply:
done